src/main/java/Curriculum.java

```diff
@@ -0,0 +1,17 @@
+import java.util.ArrayList;
+
+public class Curriculum {
+
+    static ArrayList<CourseData> curriculum = new ArrayList<>();
+    public void addCourse(CourseData courseData) {
+       curriculum.add(courseData);
+    }
+
+    public int getDuration() {
+        return curriculum.stream().mapToInt(d -> d.duration).sum();
+    }
+
+    public void clearArrayList() {
```

Comment:
why do we need this method? pls remove

Reply:
without this method curriculum of next student will be added to previous result, I implement this method then in main method line 51

Reply:
> without this method curriculum of next student will be added to previous result
could you fix this? when a new `Curriculum` object is created it shouldn't contain courses from previous students

Reply:
> without this method curriculum of next student will be added to previous result

could you fix this? when a new Curriculum object is created it shouldn't contain courses from previous students

Reply:
fixed, static in ArrayList was the reason

Reply:
great!